library/aes.c

```diff
@@ -981,13 +981,16 @@ int mbedtls_internal_aes_decrypt(mbedtls_aes_context *ctx,
  */
 MBEDTLS_MAYBE_UNUSED static void aes_maybe_realign(mbedtls_aes_context *ctx)
 {
+#if defined(MAY_NEED_TO_ALIGN)
```

Comment:
Not possible to guard the whole function, thus to move this guard line 981?

Reply:
No problem, I was copying how it was done in `mbedtls_aes_rk_offset`. I have now changed it as you suggest. PTAL, thanks.